chall09/dalba-de.c

```diff
@@ -0,0 +1,189 @@
+#include <stdio.h>
+#include <string.h>
+#include <stdlib.h>
+
+typedef struct          	s_time
+{
+  	unsigned long long		year;
+	unsigned long long		day;
+	unsigned long long		hour;
+	unsigned long long		minute;
+	unsigned long long 		second;
+	char					*str;
+	char					*cyear;
+	char					*cday;
+	char					*chour;
+	char					*cminute;
+	char					*csecond;
```

Comment:
This is a lot of elements for the same thing, you could have built the string whilst parsing to avoir allocating structures.